### PR TITLE
use `browser-sprite``browser-symbol` for `electron-renderer` target

### PR DESCRIPTION
**Pull request by meowtec**
target `electron-renderer` should be treated as `web`.

### CODE REVIEW

File: lib/configurator.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const extractTextPluginCompilerName = 'extract-text-webpack-plugin';
 const isomorphicSpriteModule = 'svg-sprite-loader/runtime/sprite.build';
 const isomorphicSymbolModule = 'svg-baker-runtime/symbol';
 
+const isTargetBrowser = target => target === 'web' || target === 'electron-renderer';
```
Comment (kisenka) [inline lib/configurator.js]: There is also `electron-main` target. I think easiest way is to test `/node/` regexp against the target string.

Comment (meowtec) [inline lib/configurator.js]: But `electron-main` is like `node`, right?

Comment (kisenka) [inline lib/configurator.js]: You're right, sorry :o)

**Comment by kisenka** [comment]
Shipped with v2.0.6

**Pull request merged**